packages/core/src/lib/LinkTo/types.ts

```diff
@@ -1,10 +1,12 @@
+import { TypographyProps } from "../Typography";
+
 export const layoutState = ['leading', 'trailing', 'none'] as const;
 export type TLayoutState = typeof layoutState[number];
 
 export const typeState = ['email', 'external', 'internal'] as const;
 export type TTypeState = typeof typeState[number];
 
-export interface LinkToProps {
+export interface LinkToProps extends TypographyProps {
```

Comment:
@0xTijan sorry but found one more bug

All props of Typography cannot be used here you have to Pick the types that are being used..
For eg: children, copyable,iconSize,onCopy  -> these are either not being used or not needed to be added to linkTo imo

Reply:
Something like this would be better 🙌🏻 
```
type TTextProps = Pick<
    TypographyProps,
    'color' | 'variant' | 'italic' | 'monospace' | 'weight'
>;
export interface LinkToProps extends TTextProps {
```

Reply:
Maybe It would be a nice idea to pass children too 🤔 
for eg -> cases where an image or some custom component needed to be used as a link, what do you think? (Not needed in this MR just a thought 🙌🏻 )